src/components/DropdownMenu.tsx

```diff
@@ -0,0 +1,59 @@
+import { useMemo } from 'react';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+import { peopleFromServer } from '../data/people';
+
+type Props = {
+  onSelect: (person: Person) => void;
+  appliedQuery: string;
+};
+
+function getFilteredPeople(people: Person[], query: string): Person[] {
+  const filteredPeople = [...people].filter(person => {
```

Comment:
```suggestion
  const filteredPeople = people.filter(person => {
```
Filter method returns a new array, you don't need to copy